assignment/assignment.hs

```diff
@@ -52,8 +59,10 @@ count element list = 0 -- replace me
 -- help to write it as two functions. (Hint: you only
 -- need to traverse the list once)
 
-evensEqualOdds :: [ Integer ] -> Bool
-evensEqualOdds numbers = False -- replace me
+--evensEqualOdds :: [ Integer ] -> Bool
+evensEqualOdds numbers = if adder numbers==0 then True else False
+adder []=0  
+adder (h:s:xs) = h-s + adder xs
 
```

Comment:
Very nice: this is the tidiest solution I've seen. My only slight niggle is with the name `adder`, but I agree it isn't an easy function to name (my version had `differenceOfPairs` but that's not ideal, either)